packages/doc/docs/api-building/configuration/data-source-profile.mdx

```diff
@@ -0,0 +1,84 @@
+# Data Source Profile
+
+When sending an API request to query the data through SQL file logistic, also should check the data source ( e.g: Database, Warehouse ) has connected, so VulcanSQL needs you to define the Data Source Profile.
+
+# Profile Structure
+
+The profile could define in two ways:
+
+## Define profile in independent YAML files
+
+You could create independent YAML files to define data sources and set the filePath in the `profiles` options of the project config:
+
+```yaml
+# pg-profile.yaml
+- name: pg
+  type: postgres
+  allow: '*'
+  connection:
+    host: example.com
+    user: vulcan
+    password: secret
+    database: vulcan
+    port: 5432
+
+------------------------------------
+# duck-profile.yaml
+name: duck
+type: duckdb
+connection:
+   persistent-path: ./test-data/moma.db
+   log-queries: true
+   log-parameters: true
+allow: '*'
+
+-------------------------
+# vulcan.yaml (project config)
+extensions:
+  postgres: '@vulcan-sql/extension-driver-duckdb'
+  duckdb: '@vulcan-sql/extension-driver-duckdb'
+
+profiles:
+  - ./pg-profile.yaml
+  - ./duck-profile.yaml
+```
+
+You should make sure you have installed the data source driver package and declared the module name under the `extensions` of the project config.
+
+## Define profile in the project config
+
+Or you could define the data source profile under `profiles` options of the project config :
+
+```yaml
+# vulcan.yaml (project config)
+extensions:
+  duckdb: '@vulcan-sql/extension-driver-duckdb'
+  postgres: '@vulcan-sql/extension-driver-duckdb'
+
+profiles:
+  - name: duck
+    type: duckdb
+    connection:
+      persistent-path: ./test-data/moma.db
+      log-queries: true
+      log-parameters: true
+    allow: '*'
+  - name: pg
+    type: postgres
+    allow: '*'
+    connection:
+      host: example.com
+      user: vulcan
+      password: secret
+      database: vulcan
+      port: 5432
+```
```

Comment:
We haven't supported defining profiles in project config~

Reply:
Sorry, I misunderstood your PR #75 description, could you assist me to remove this  section ? Thanks